Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractRandomCutTree.java

```diff
@@ -100,12 +100,16 @@ protected AbstractRandomCutTree(AbstractRandomCutTree.Builder<?> builder) {
         }
     }
 
+    protected void changeCacheFraction(double fraction) {
```

Comment:
I think we don't need this method (see commend on AbstractCompactRandomCutTree).

Reply:
Done